django_celery_monitor/managers.py

```diff
@@ -39,3 +92,23 @@ def purge(self):
                 'DELETE FROM {0.db_table} WHERE hidden=%s'.format(meta),
```

Comment:
I'm confused why you use raw sql here? What's wrong with `self.model.objects.using(router.db_for_write(self.model)).filter(hidden=True).delete()`?